service/history/conflictResolver_test.go

```diff
@@ -58,7 +58,7 @@ type (
 		mockTxProcessor          *MocktransferQueueProcessor
 		mockReplicationProcessor *MockReplicatorQueueProcessor
 		mockTimerProcessor       *MocktimerQueueProcessor
-		mockEventsCache          *events.MockCache
+		mockEventsCache          *cache.MockCache
```

Comment:
I believe `*events.MockCache` and `*cache.MockCache` are mocking two different interfaces and I think we need to use the first one.

Reply:
yep, it is a draft :)